spec/tree_spec.rb

```diff
@@ -1,14 +1,63 @@
 require 'rspec'
 require 'tree'
 
+tree = Tree.new
+
 describe 'Tree' do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(Tree_class.is_a? 'Class').to be_true
```

Comment:
Do your tests pass when you run rspec?

https://ruby-doc.org/core-2.4.1/Object.html#method-i-is_a-3F